include/ethernet.h

```diff
@@ -0,0 +1,92 @@
+/*
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *
+ *  Copyright (C) 2021  The DOSBox Team
+ *
+ *  This program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 2 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License along
+ *  with this program; if not, write to the Free Software Foundation, Inc.,
+ *  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+ */
+
+#ifndef DOSBOX_ETHERNET_H
+#define DOSBOX_ETHERNET_H
+
+#include <functional>
+
+#include "config.h"
```

Comment:
This is a mistake and it is going to result in broken builds on Windows or other non-Linux platform (sooner or later).

We follow Google C++ coding style include order: https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes to avoid these compilation issues.

In this case correct include order should be:

```
#include "dosbox.h"

#include <functional>
#include <string>
```

`dosbox.h` and `config.h` are our two main headers - in this case you want `dosbox.h` to pull-in forward declaration of Section pointer and inttypes.

Reply:
Fixed.